apps/dev/outstatic/content/metadata.json

```diff
@@ -0,0 +1,127 @@
+{
+  "commit": "a3e888593a592fe5cbabf1957da818fca1f4147b",
+  "generated": "Wed, 28 Dec 2022 09:00:19 GMT",
+  "metadata": [
+    {
+      "title": "Learn how to create your blog with Outstatic",
+      "status": "published",
+      "description": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore.",
+      "coverImage": "/images/jupiter-spaceship.png",
+      "publishedAt": "2022-07-14T02:34:02.322Z",
+      "author": {
+        "name": "Andre Vitorio"
+      },
+      "slug": "how-to-create-an-outstatic-blog",
+      "__outstatic": {
+        "commit": "bf4993b965319da4cb9de4a3810fb4af12bdebd6",
+        "hash": "2596208214",
+        "path": "apps/dev/outstatic/content/posts/how-to-create-an-outstatic-blog.md"
+      },
+      "category": "posts"
+    },
+    {
+      "title": "Home",
+      "status": "published",
+      "author": {
+        "name": "Andre Vitorio",
+        "picture": ""
+      },
+      "slug": "home",
+      "description": "This is my blog.",
+      "coverImage": "",
+      "publishedAt": "2022-03-17T05:35:07.322Z",
+      "__outstatic": {
+        "commit": "2e47122b5b6d49d6d1ddb15bcd33edc4a0c1256e",
+        "hash": "217348983",
+        "path": "apps/dev/outstatic/content/pages/home.md"
```

Comment:
I'm getting an error on the frontend locally when visiting the dev blog. 

```
Server Error
Error: ENOENT: no such file or directory, open '/Users/username/site/outstatic/dev/apps/dev/outstatic/content/apps/dev/outstatic/content/pages/home.md'
```

Let me know if this is happening with you. I've managed to fix it by adding replacing the `rootPath` on the `o.path` variable for an empty string.

I wonder if we should be fetching from Github at all times, even when getting the .mds on the frontend/dev.

Reply:
> I wonder if we should be fetching from Github at all times, even when getting the .mds on the frontend/dev.

I thought about that, but then it wouldn't really be a static site CMS anymore. :)

Regarding, the error, I brought everything up successfully on my side, but I do remember seeing the duplicate path error before. The core issue is that from github we see the full file path to the markdown file, which is what we save. `CONTENT_PATH` is defined as

```ts
const CONTENT_PATH = join(
  process.cwd(),
  process.env.OST_CONTENT_PATH || 'outstatic/content'
)
```

The fix is probably something with `OST_MONOREPO_PATH`